client/landing/jetpack-cloud/sections/backups/index.js

```diff
@@ -11,16 +11,25 @@ import config from 'config';
 import { navigation, siteSelection, sites } from 'my-sites/controller';
 import { makeLayout, render as clientRender } from 'controller';
 import {
-	backups,
+	backupActivityLog,
 	backupDetail,
 	backupDownload,
 	backupRestore,
+	backups,
 } from 'landing/jetpack-cloud/sections/backups/controller';
 
 export default function() {
 	if ( config.isEnabled( 'jetpack-cloud/backups' ) ) {
-		page( '/backups', siteSelection, sites, navigation, makeLayout, clientRender );
-		page( '/backups/:site', siteSelection, navigation, backups, makeLayout, clientRender );
+		page( '/backups/activity', siteSelection, sites, makeLayout, clientRender );
+		page(
+			'/backups/activity/:site',
```

Comment:
I prefer to follow the same route pattern as the rest pages `/backups/:site/{sub-section}`. When we merge your proposal on https://github.com/Automattic/wp-calypso/pull/40630 then it will follow this pattern with its own path function.

Reply:
The reason why this route does not follow the sub section logic of the the sections is that it is possible to navigate here without a site ( from the sidebar, for example ) and that navigating here without a site makes sense ( since we don't need anymore information once we get here, as opposed to restore, download, or detail which all need another id to display anything )

With that stated,  [the `siteSelection` function we use from Calypso](https://github.com/Automattic/wp-calypso/blob/5f83029a8074db20672622920904ebee388aa914/client/my-sites/controller.js#L320) does not have sophisticated logic when a site is picked: [It is literally just added to the end up the existing path](https://github.com/Automattic/wp-calypso/blob/5f83029a8074db20672622920904ebee388aa914/client/my-sites/controller.js#L358).

So, to sum up, when using a route that is compatible with siteSelection we need to be able to just append the siteSlug to the end, hence the `backups/activity` approach